app/classes/mappable/box_methods.rb

```diff
@@ -66,6 +66,21 @@ def east_west_distance
       west <= east ? east - west : east - west + 360
     end
 
+    # Converts degrees to radians.
+    def to_radians(degrees)
```

Comment:
Would it make sense to instead extend Numeric to include a `radians` method?
Then you could write `degrees.radians`.

Reply:
I like this idea. Will investigate. (Hopefully we already extend `Numeric`)